src/main/java/org/myrobotlab/service/meta/TrackingMeta.java

```diff
@@ -23,49 +21,63 @@
    * This class is contains all the meta data details of a service. It's peers,
    * dependencies, and all other meta data related to the service.
    * 
-   * @param name
+   * @param type
    *          n
    * 
    */
-  public TrackingMeta(String name) {
+  public TrackingMeta() {
 
-    super(name);
-    Platform platform = Platform.getLocalInstance();
+    addPeer("cv", "OpenCV");
+    addPeer("pan", "Servo");
```

Comment:
how is this going to play with users who want to define their own servo services for pan and tilt?

Reply:
working on that